CHANGELOG.md

```diff
@@ -24,6 +24,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [tripleCC](https://github.com/tripleCC)
   [#7958](https://github.com/CocoaPods/CocoaPods/issues/7958)
 
+* Fix crash when exec pod update with --sources and --project-directory  
```

Comment:
nit: replace "exec" with "running" and wrap \`pod update\` with backticks as well as \`--sources\` and \`--project-directory\`

Reply:
done